imports/client/components/FirehosePage.tsx

```diff
@@ -0,0 +1,267 @@
+import { Meteor } from 'meteor/meteor';
+import { useTracker } from 'meteor/react-meteor-data';
+import { _ } from 'meteor/underscore';
+import { faEraser } from '@fortawesome/free-solid-svg-icons/faEraser';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, {
+  useCallback, useEffect, useMemo, useRef, useState,
+} from 'react';
+import Button from 'react-bootstrap/Button';
+import FormControl, { FormControlProps } from 'react-bootstrap/FormControl';
+import FormGroup from 'react-bootstrap/FormGroup';
+import InputGroup from 'react-bootstrap/InputGroup';
+import { RouteComponentProps } from 'react-router';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { shortCalendarTimeFormat } from '../../lib/calendarTimeFormat';
+import ChatMessages from '../../lib/models/chats';
+import Profiles from '../../lib/models/profiles';
+import Puzzles from '../../lib/models/puzzles';
+import { ChatMessageType } from '../../lib/schemas/chat';
+import { PuzzleType } from '../../lib/schemas/puzzle';
+import { useBreadcrumb } from '../hooks/breadcrumb';
+import FixedLayout from './styling/FixedLayout';
+
+const FirehosePageLayout = styled.div`
+  padding: 8px 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  max-height: calc(100vh - 60px);
```

Comment:
This lands inside of a `.FixedLayout` that takes care of keeping clear of the Nav, so I don't see any reason to have this calc here (and eliminating it is an improvement to modularity).

Whether or not `max-height` is specified, I think the chat log should fill the available space vertically.

This suggestion addresses both:
```suggestion
  height: 100%;
```

Reply:
You're right, this is better.  Incorporated into the latest rev.